apps/contract-sync-job/src/db-helpers.ts

```diff
@@ -0,0 +1,73 @@
+import { GovrnProtocol } from '@govrn/protocol-client';
+
+export const getOrInsertUser = async (
+  govrn: GovrnProtocol,
+  data: { address: string },
+): Promise<number> => {
+  // Query existing user.
+  const user = await govrn.user.list({
+    where: {
+      address: { equals: data.address },
+    },
+    first: 1,
+  });
+
+  // Returns the id, if user exists.
+  if (user.length === 1) {
+    return user[0]?.id;
+  }
+
+  // Insert new user into db, if user doesn't exist.
+  const newUser = await govrn.user.create({
+    address: data.address,
+    username: data.address,
+  });
+
+  return newUser.id;
+};
+
+export const getOrInsertActivityType = async (
+  govrn: GovrnProtocol,
+  data: { name: string },
+): Promise<number> => {
+  // Check for an existing activity types.
+  const activityTypeId = await govrn.activity_type.list({
```

Comment:
Same here is get possible over list?

Reply:
`ActivityTypes` Resolvers doesn't implement a `get()` function. 
Similar to `user` we query by `name`.